### PR TITLE
Add heroku support.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
All you need to run sugardough based projects on heroku is a Procfile and moving around some things. 
@pmclanahan @jgmize r?

File: {{ cookiecutter.project_name }}/bin/run-common.sh [hunk 1]
```diff
@@ -1,5 +1,3 @@
 #!/bin/bash
 
-./manage.py collectstatic --noinput -c
 ./manage.py syncdb --noinput
```
Comment (pmclanahan) [inline {{ cookiecutter.project_name }}/bin/run-common.sh]: Isn't `syncdb` deprecated in favor of `migrate` since 1.7?

**Comment by pmclanahan** [comment]
Very cool! Seems quite the reasonable thing to do.

File: {{ cookiecutter.project_name }}/bin/run-common.sh (hunk 1 above)
Comment (glogiotatidis) [inline {{ cookiecutter.project_name }}/bin/run-common.sh]: Indeed! I'm stuck with south i guess :)

**Comment by glogiotatidis** [comment]
Thanks @pmclanahan!

**Pull request merged**